mmf/utils/metsumm.py

```diff
@@ -0,0 +1,8 @@
+def metsumm(stepno=''): 
```

Comment:
This should go into general utils.

Reply:
A teammate suggested that this function is not mmf specific and likely should be part of main xla library. So removed it for the time being. If we see the need we can add it as you suggested.